src/Runner.Worker/ExecutionContext.cs

```diff
@@ -149,6 +156,29 @@ public sealed class ExecutionContext : RunnerService, IExecutionContext
         private long _totalThrottlingDelayInMilliseconds = 0;
         private bool _stepTelemetryPublished = false;
 
+        public ExecutionContext()
+            : this(null, false)
```

Comment:
```suggestion
            : this(parent: null, embedded: false)
```
maybe ☝️ for better readability?